src/jon_mikkel_ex/ex04/myrand.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Jon-Mikkel Korsvik"
+__email__ = "jonkors@nmbu.no"
+
+from pprint import pprint
+
+
+class LCGRand:
+    def __init__(self, seed):
+        """
+
+        :param seed: A number selected when initiliazing, integer, set into
+        a list.
+        a: is a large odd number
+        m: is a large odd number
+        """
+        self.seed = [seed]
+        self.a = 7**5
+        self.m = 2**31-1
+        self.idx = 0
+
+    def rand(self):
+        """
+        Index (idx) is updated each time rand is called. The seed list is
+        appended with the newly calculated "random number" based on the
+        first seed.
+        :return: the last seed from the list. list of integers
+        """
+        self.idx += 1
+        self.seed.append((self.a * self.seed[self.idx - 1]) % self.m)
+        return self.seed[self.idx]
```

Comment:
Since you are appending to the list, you could get the last element instead of the current one. i.e. `return seed[-1]`.
Cool approach storing all the values.